src/components/Typography/HX/HX.stories.tsx

```diff
@@ -0,0 +1,154 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { HX } from './HX'
+import { Icon } from '../../Icon'
+import { Typography } from '..'
+import { loremIpsum } from '../../../utils/loremIpsum'
+
+const customEllipsis = {
+  rows: 1,
+  suffix: ' END',
+  expandable: true,
+  symbol: 'Show more',
+}
+
+const customCopyable = {
+  text: 'Custom text to copy',
+  icon: [
+    <Icon
+      color="magenta"
+      key="FiCopy"
+      name="FiCopy"
+      size={16}
+    />,
+    <Icon
+      color="magenta"
+      key="FiCheck"
+      name="FiCheck"
+      size={16}
+    />,
+  ],
+  tooltips: ['Copy to clipboard!', 'Copied to clipboard!'],
+}
+
+const meta = {
+  title: 'Components/Typography/HX',
+  component: HX,
+  args: {
+    ...HX.defaultProps,
+    children: 'Text',
+  },
+} satisfies Meta<typeof HX>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const HXComponent: Story = {
+  args: { ...meta.args, level: 1 },
+}
+
+export const All = {
+  argTypes: {
+    children: { table: { disable: true } },
+    level: { table: { disable: true } },
```

Comment:
I would remove the level declaration from every story and add it to meta argTypes. Then I would enable it specifically for the HXComponent story